File: C/Exercicio14.c
#include <stdio.h>
 
int main(int argc, char *argv[]){
  int val1, val2, val3;
  
  printf("Digite o primeiro valor: ");
  scanf("%i", &val1);
 
  printf("Digite o segundo valor: ");
  scanf("%i", &val2);

  printf("Digite o terceiro valor: ");
  scanf("%i", &val3);
 
  if (val1 > val2){
      if (val1 > val3){
        printf("O maior valor e o primeiro: %i", val1);
      }
  }

  if (val2 > val1){
      if (val2 > val3){
        printf("O maior valor e o segundo: %i", val2);
      }
  }

  if (val3 > val2){
      if (val3 > val1){
        printf("O maior valor e o terceiro: %i", val3);
      }
  }

  return 0;
}

File: Python/Exercicio14.py
val1 = int(input("Digite o primeiro valor: "))
val2 = int(input("Digite o segundo valor: "))
val3 = int(input("Digite o terceiro valor: "))

if (val1>val2) and (val1>val3):
    print("O maior valor é:", val1)
if (val2>val1) and (val2>val3):
    print("O maior valor é:", val2)
if (val3>val1) and (val3>val2):
    print("O maior valor é:", val3)